src/diffusers/models/attention.py

```diff
@@ -364,3 +365,142 @@ def forward(self, x, emb):
         x = F.group_norm(x, self.num_groups, eps=self.eps)
         x = x * (1 + scale) + shift
         return x
+
+
+class BasicSparseTransformerBlock(nn.Module):
```

Comment:
Why is it called "Sparse" ?

Reply:
The original authors refer to it as Sparse Causal Attention, so I named it `BasicSparseTransformerBlock`to differentiate from `BasicTransformerBlock`.